airgun/views/contenthost.py

```diff
@@ -32,6 +32,7 @@
     EditableEntrySelect,
     ReadOnlyEntry,
     Search,
+    ActionDropdownWithCheckbox
```

Comment:
you can sort that alphabetically in case you will do anything else in that PR =)